docs/source/testing/scenarios.rst

```diff
@@ -3,16 +3,135 @@
 
 Test Scenarios
 ==============
+Scenarios are distinct environments that describe a Ceph deployment and
+configuration. Scenarios are isolated as well, and define what machines are
+needed aside from any ceph-ansible configuration.
 
 .. _scenario_files:
 
 Scenario Files
---------------
+==============
+The scenario is described in a ``vagrant_variables.yml`` file, which is
+consumed by ``Vagrant`` when bringing up an environment.
+
+This yaml file is loaded on the ``Vagrantfile`` so that the settings can be
```

Comment:
I want to load it on to it
![flav](https://cloud.githubusercontent.com/assets/317847/24979817/a7546476-1fa3-11e7-8566-f1dc39a07248.gif)